src/amo/components/AddonMoreInfo.scss

```diff
@@ -1,5 +1,5 @@
 @import "~core/css/inc/mixins";
-@import "~core/css/inc/vars";
+@import "~amo/css/inc/vars";
```

Comment:
I guess I could have updated the core styles but the base font colour and masthead colour didn't feel like they belonged in core. The amo vars import the core vars so common stuff can still go in core and the import can be from amo.